konan/konan.properties

```diff
@@ -343,63 +343,88 @@ dynamicLinker.linux_mipsel32 = /lib/ld.so.1
 abiSpecificLibraries.linux_mipsel32 =
 
 # Android ARM32, based on NDK for android-21.
-targetToolchain.macos_x64-android_arm32 = target-toolchain-21-osx-android_arm32
-# TODO: split dependencies to host-dependent and host-independent parts.
+targetToolchain.macos_x64-android_arm32 = target-toolchain-1-osx-android_ndk
 dependencies.macos_x64-android_arm32 = \
     clang-llvm-6.0.1-darwin-macos \
-    target-sysroot-21-android_arm32 \
-    target-toolchain-21-osx-android_arm32 \
+    target-sysroot-1-android_ndk \
+    target-toolchain-1-osx-android_ndk \
     libffi-3.2.1-3-darwin-macos
-targetToolchain.linux_x64-android_arm32 = target-toolchain-21-linux-android_arm32
+targetToolchain.linux_x64-android_arm32 = target-toolchain-1-linux-android_ndk
 dependencies.linux_x64-android_arm32 = \
     clang-llvm-6.0.1-linux-x86-64 \
-    target-sysroot-21-android_arm32 \
-    target-toolchain-21-linux-android_arm32 \
+    target-sysroot-1-android_ndk \
+    target-toolchain-1-linux-android_ndk \
     libffi-3.2.1-2-linux-x86-64
-targetToolchain.mingw_x64-android_arm32 = target-toolchain-21-windows-android_arm32
+targetToolchain.mingw_x64-android_arm32 = target-toolchain-1-windows-android_ndk
 dependencies.mingw_x64-android_arm32 = \
     msys2-mingw-w64-x86_64-gcc-7.3.0-clang-llvm-lld-6.0.1-2 \
-    target-sysroot-21-android_arm32 \
-    target-toolchain-21-windows-android_arm32 \
+    target-sysroot-1-android_ndk \
+    target-toolchain-1-windows-android_ndk \
     libffi-3.2.1-mingw-w64-x86-64
 
 quadruple.android_arm32 = arm-linux-androideabi
-entrySelector.android_arm32 =  -Wl,--defsym -Wl,main=Konan_main
+entrySelector.android_arm32 = -Wl,--defsym -Wl,main=Konan_main
 clangFlags.android_arm32 = -cc1 -target-cpu arm7tdmi -emit-obj -disable-llvm-optzns -x ir -femulated-tls -mrelocation-model pic
 clangOptFlags.android_arm32 = -O3 -ffunction-sections
 linkerNoDebugFlags.android_arm32 = -Wl,-S
 clangNooptFlags.android_arm32 = -O1
-targetSysRoot.android_arm32 = target-sysroot-21-android_arm32
-linkerKonanFlags.android_arm32 = -lm -latomic -lstdc++ -landroid
+targetSysRoot.android_arm32 = target-sysroot-1-android_ndk
+linkerKonanFlags.android_arm32 = -lm -lstdc++ -landroid -llog -latomic
 
-# Android ARM64, based on NDK for android-21.
-targetToolchain.macos_x64-android_arm64 = target-toolchain-21-osx-android_arm64
-# TODO: split dependencies to host-dependent and host-independent parts.
-dependencies.macos_x64-android_arm64 = \
+# Android ARM64, based on NDK.
+targetToolchain.macos_x64-android_arm64 = target-toolchain-1-osx-android_ndk
+dependencies.macos_x64-android_x64 = \
```

Comment:
`android_x64` and `android_arm64` seem mixed up.

Reply:
Yeah, wrong order.